app/main/models/enums.py

```diff
@@ -0,0 +1,20 @@
+"""
+All enums used in the schema are described here. All models should refer
```

Comment:
Good. We do not need to write "all models...." Just the first sentence is enough.

Even better is using, "The module contains all the Enums"